Dockerfile.alpine

```diff
@@ -0,0 +1,83 @@
+FROM ekidd/rust-musl-builder AS cargo-audit-build
+
+RUN cargo install cargo-audit --root /home/rust && \
+    strip /home/rust/bin/cargo-audit
+
+FROM alpine:3.10
+
+ENV RUBY_VERSION=2.6.3
+ENV FINDSECBUGS_VERSION=1.8.0
+ENV OWASP_VERSION=5.0.0
+
+ARG FINDSECBUGS_FOLDER=/usr/local/opt/findsecbugs
+ARG OWASP_DEP_FOLDER=/usr/local/bin/owaspdependency
+
+RUN apk update  && \
+    apk add  bash \
+    && bash --login
+RUN apk add ca-certificates \
+    openssl openssl-dev \
+    alpine-sdk gcc g++ make \
+    libffi-dev zlib zlib-dev \
+    gnupg procps musl-dev sqlite \
+    linux-headers shadow \
+    nodejs npm yarn \
+    openjdk8-jre maven \
+    python python-dev \
+    perl git \
+    php7 php7-cli \
+    pcre-dev \
+    && curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" \
+    && python get-pip.py \
+    && pip install safety==1.8.4 piprot==0.9.10 bandit==1.5.1
+
+RUN curl -sSL https://github.com/rvm/rvm/tarball/stable -o rvm-stable.tar.gz \
+    && echo 'export rvm_prefix="$HOME"' > /root/.rvmrc \
+    && echo 'export rvm_path="$HOME/.rvm"' >> /root/.rvmrc \
+    && mkdir rvm && cd rvm \
+    && tar --strip-components=1 -xzf ../rvm-stable.tar.gz \
+    && ./install --auto-dotfiles --autolibs=0 \
+    && source ~/.rvm/scripts/rvm \
+    && /bin/bash -l -c "rvm requirements" \
+    && /bin/bash -l -c "rvm autolibs 4" \
+    && yes | /bin/bash -l -c "rvm install ${RUBY_VERSION}" \
+    && /bin/bash -l -c "gem install bundler:2.0.1 bundler-audit:0.6.1 brakeman:4.5.1" \
+    && /bin/bash -l -c "bundle audit update"
+
+RUN mkdir -p ${FINDSECBUGS_FOLDER} && cd ${FINDSECBUGS_FOLDER} \
+    && wget --quiet https://github.com/find-sec-bugs/find-sec-bugs/releases/download/version-${FINDSECBUGS_VERSION}/findsecbugs-cli-${FINDSECBUGS_VERSION}.zip \
+    && unzip -q findsecbugs-cli-${FINDSECBUGS_VERSION}.zip \
+    && rm findsecbugs.sh \
+    && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+ENV PATH $HOME/.cargo/bin:$PATH
+
+COPY scripts/findsecbugs.sh ${FINDSECBUGS_FOLDER}/findsecbugs.sh
+
+COPY --from=cargo-audit-build /home/rust/bin/cargo-audit /usr/local/bin/
+
+RUN chmod +x ${FINDSECBUGS_FOLDER}/findsecbugs.sh \
+    && ln -s ${FINDSECBUGS_FOLDER}/findsecbugs.sh /usr/local/bin/findsecbugs \
+    && mkdir $OWASP_DEP_FOLDER && cd $OWASP_DEP_FOLDER \
+    && wget --quiet http://dl.bintray.com/jeremy-long/owasp/dependency-check-${OWASP_VERSION}-release.zip \
+    && unzip -q dependency-check-${OWASP_VERSION}-release.zip \
+    && chmod +x $OWASP_DEP_FOLDER/dependency-check/bin/dependency-check.sh \
+    && rm dependency-check-${OWASP_VERSION}-release.zip \
+    && mv dependency-check/bin/dependency-check.sh dependency-check/bin/dependency-check \
```

Comment:
The OWASP scan is also failed because no database exists. I believe this is because missing the 'dependency-check --updateonly' command to create the OWASP database in the image.